vpr/src/place/analytical_placement/PartialPlacement.cpp

```diff
@@ -20,12 +20,16 @@ PartialPlacement::PartialPlacement(const AtomNetlist& netlist,
 
     const AtomContext& atom_ctx = g_vpr_ctx.atom();
 
+    std::vector<std::unordered_set<t_pack_molecule*>> interesting_nets;
+    interesting_nets.reserve(atom_netlist.nets().size());
```

Comment:
Interesting name :)

Reply:
Do you think it should be changed?

Reply:
It is a temporary data structure for filtering, I don't see why it is interesting, but if we plan on using the vtr netlist in the future, it would not matter that much now.